node/pkg/driver/device_connectivity/device_path_s390x.go

```diff
@@ -0,0 +1,5 @@
+package device_connectivity
+
+// https://cgit.freedesktop.org/systemd/systemd/tree/src/udev/udev-builtin-path_id.c#n613
+// on Z-Systems, the udev subsystem is a Channel Command Word
+var FcSubsystem = "ccw"
```

Comment:
use const.
I think it should be private per package (use first letter in lowercase)

Reply:
![image](https://user-images.githubusercontent.com/57443811/75152459-bae74280-5711-11ea-9b9b-aeb3dac95b67.png)